mix.exs

```diff
@@ -2,15 +2,23 @@ defmodule Changelog.Mixfile do
   use Mix.Project
 
   def project do
-    [app: :changelog,
-     version: "0.0.1",
-     elixir: "~> 1.3",
-     elixirc_paths: elixirc_paths(Mix.env),
-     compilers: [:phoenix] ++ Mix.compilers,
-     build_embedded: Mix.env == :prod,
-     start_permanent: Mix.env == :prod,
-     aliases: aliases,
-     deps: deps]
+    [
+      app: :changelog,
+      version: "0.0.1",
+      elixir: "~> 1.3",
+      elixirc_paths: elixirc_paths(Mix.env),
+      compilers: [:phoenix] ++ Mix.compilers,
+      build_embedded: Mix.env == :prod,
+      start_permanent: Mix.env == :prod,
+      aliases: aliases,
+      deps: deps,
+      preferred_cli_env: [
```

Comment:
Most of this is just indentation. 

Personally I find the single space indent of the generated file annoying when going to add something like `preferred_cli_env` that is further indented.